### PR TITLE
Visual element packager

### PR DESCRIPTION
**Pull request by Clancey**
### Description of Change ###
Fixed a crash with null renderers. Or invalid indexes
Jason Smith should approve!
### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Platform.Android/VisualElementPackager.cs [hunk 1]
```diff
@@ -159,7 +160,7 @@ void OnChildAdded(object sender, ElementEventArgs e)
 			if (view != null)
 				AddChild(view);
 
-			if (ElementController.LogicalChildren[ElementController.LogicalChildren.Count - 1] != view)
+			if (ElementController.LogicalChildren.FirstOrDefault() != view)
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Platform.Android/VisualElementPackager.cs]: we try not to use linq expressions

Comment (Clancey) [reply Xamarin.Forms.Platform.Android/VisualElementPackager.cs]: I understand. But in the case it is correct. Safety is better.

Comment (myroot) [reply Xamarin.Forms.Platform.Android/VisualElementPackager.cs]: Why not **Last**orDefault?

Comment (Clancey) [reply Xamarin.Forms.Platform.Android/VisualElementPackager.cs]: You are correct. It should be last or default.

Comment (StephaneDelcroix) [reply Xamarin.Forms.Platform.Android/VisualElementPackager.cs]: well, that will enumerate all the children for nothing. probably why we used the indexer before...

Comment (myroot) [reply Xamarin.Forms.Platform.Android/VisualElementPackager.cs]: Actually, did not enumerate all children, If it's instance of IList

**Pull request merged**